compiler/bootstrap/compilation/x64/64/Holmakefile

```diff
@@ -1,36 +1,16 @@
 ARCH = x64
 WORD_SIZE = 64
-INCLUDES = ../../../translation ../../../../../semantics ../../../.. ../../../../backend/$(ARCH) ../../../../encoders/asm ../../../../encoders/$(ARCH) ../../../../../basis
-CLINE_OPTIONS = --qof
+INCLUDES = $(CAKEMLDIR)/semantics $(CAKEMLDIR)/basis ../../../..\
```

Comment:
Define `../../../../` in a Holmakefile variable (perhaps wrt `CAKEMLDIR` rather than current directory), and then reuse?

Reply:
I've been following the convention that directories under the same top-level directory don't get the CAKEMLDIR treatment. And in this case, `$(CAKEMLDIR)/compiler` isn't shorter than `../../../..` so I'm going to leave it as is. (A separate shorter variable could be nice, but not worth it in my opinion.)